src/wp-includes/class-wp-object-cache.php

```diff
@@ -323,6 +323,67 @@ public function get_multiple( $keys, $group = 'default', $force = false ) {
 		return $values;
 	}
 
+	/**
+	 * Delete multiple values from the cache in one call.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param array  $keys  Array of keys to be delete.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @return array Array of return values organized into groups.
+	 */
+	public function delete_multiple( $keys, $group = '' ) {
```

Comment:
Same as above as well.

```suggestion
	public function delete_multiple( array $keys, $group = '' ) {
```

Reply:
Actioned in https://github.com/WordPress/wordpress-develop/pull/2018